data/data.go

```diff
@@ -53,9 +53,11 @@ func setMetadata(config *effx_api.ConfigurationFile, m *metadata.Result) *effx_a
 		}
 
 		if m.Language != "" && m.Version != "" {
+			language := strings.ToLower(m.Language)
+			version := strings.ToLower(m.Version)
 			safelySetField(config.Annotations, "effx.io/inferred-tags", fmt.Sprintf("language,%s", m.Language))
```

Comment:
i think the logic here could be better a little bit. perhaps we should allow inferring one of these tags by itself, instead of requiring both. Also, we might consider not setting this field for detected services

Reply:
@weeyum So Joey mentioned a while back that having a language tag without a version is pointless. How do you feel about this? 

I just added a change that allows the language field to be set without a version. Also this function is not called for detected services.

Reply:
right, but the idea is that the logic would only be skipped if someone already defined it. so it would still have it, but i'm saying that we should allow a user to define 1 of the 2 tags

Reply:
for example

customer:
language: go

inferred
go:1.13.5

I would expect annotations to have inferred-tags:go

Reply:
@DanielMcSheehy see this comment ^

Reply:
but also we should make it work for the inverse, where they define `go:...` and we infer `language:go`

Reply:
I'm fixing this in a different PR